creme/activities/bricks.py

```diff
@@ -325,6 +327,78 @@ def detailview_display(self, context):
         ))
 
 
+class FullCalendarBrick(Brick):
+    verbose_name = _('Activities calendar')
+    dependencies = (Activity, Contact, Calendar,)
+    template_name = 'activities/bricks/activity-fullcalendar.html'
+    permissions = 'activities'
+
+    allow_event_create = False
+
+    show_headless = False
+    show_week_number = True
+    show_timezone_info = False
+
+    def get_calendar_settings(self, context):
+        return {
+            "utc_offset": get_current_utc_offset(),
+            "allow_event_create": self.allow_event_create,
+            "headless_mode": self.show_headless,
+            "show_week_number": self.show_week_number,
+            "show_timezone_info": self.show_timezone_info,
+            **CalendarConfigItem.objects.for_user(context['user']).as_dict(),
+        }
+
+    def get_event_fetch_url(self, context):
+        return reverse('activities__calendars_activities')
+
+    def get_event_update_url(self, context):
+        return reverse('activities__set_activity_dates')
+
+    def get_event_create_url(self, context):
+        return reverse('activities__create_activity_popup')
+
+    def get_calendar_sources(self, context):
+        return list(
+            Calendar.objects.filter(
+                user=context['user'],
+                is_default=True,
+            ).values_list('pk', flat=True)
+        )
+
+    def get_calendar_context(self, context):
+        return {
+            'calendar_sources': self.get_calendar_sources(context),
+            'calendar_settings': self.get_calendar_settings(context),
+            'event_fetch_url': self.get_event_fetch_url(context),
+            'event_update_url': self.get_event_update_url(context),
+            'event_create_url': self.get_event_create_url(context),
+        }
+
+
+class MyActivitiesCalendarBrick(FullCalendarBrick):
+    id = Brick.generate_id('activities', 'my_activities_calendar')
+    verbose_name = _('Activities for me')
```

Comment:
This verbose name is use in the configuration. maybe something like "My Calendar" would be more informative :thinking:

Reply:
The title is already prefixed by "Calendar −" in the template. So I copied the title of future/past activities